### ISSUE TITLE
Different icon in tray if new tweets are available?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#3]
```
I would like to the icon in the tray to notify me of new messages. E.g.
Pidgin, Mitter, Liferea etc.

This is the only reason I'm not using Pino at the moment.
```

Original issue reported on code.google.com by `58desi...@gmail.com` on 12 Dec 2009 at 7:16

**Comment by GoogleCodeExporter** [comment]
```
Good idea. Added to the TODO list.
```

Original comment by `tro...@gmail.com` on 12 Dec 2009 at 10:12
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
to see the icon*

Thanks! Once that feature is added I'll start using Pino right away.
```

Original comment by `58desi...@gmail.com` on 12 Dec 2009 at 10:18

**Comment by GoogleCodeExporter** [comment]
```
Closed by fe320a0c24 revision.
```

Original comment by `tro...@gmail.com` on 13 Dec 2009 at 12:57
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Wow, that was fast! I'm very grateful. Pino is absolutely my new favorite 
Twitter client.

Btw, I spoke with Philip5 so he has put up Pino for Ubuntu 9.10 (x86 and 64-bit
version) in his very popular PPA: 
https://edge.launchpad.net/~philip5/+archive/extra
```

Original comment by `58desi...@gmail.com` on 13 Dec 2009 at 6:37
